### DISCUSSION TITLE
Trakt not showing Movie Progress

### DISCUSSION THREAD

**Discussion by jbrines** [Taxel/PlexTraktSync#1416] [Support] [answered]
When I have watched a movie it is not showing in the Trakt progress but it does show in the history.

Why would this happen?

**Comment by simonc56** [comment, marked as answer]
I guess it's how Trakt works.
What answer did Trakt support give you about it ?

**Reply by jbrines** [reply]
Sorry I didn't ask, I just wanted tto make sure I wasn't doing anything wrong this PlexTraktSync first
